archinstall/lib/user_interaction.py

```diff
@@ -939,3 +939,86 @@ def select_locale_enc(default):
 	).run()
 
 	return selected_locale
+
+def generic_select(p_options :Union[list,dict],
+				input_text :str = "Select one of the above by index or absolute value: ",
```

Comment:
Can we change the "above" to "below" because the options are under the text :) (some for the multi select function)

Reply:
You're totally right ;-) And, there is no index anymore (I grew with IBM's CUA and miss them)